src/com/jwetherell/algorithms/mathematics/Division.java

```diff
@@ -22,6 +28,9 @@ public static final long divisionUsingLoop(int a, int b) {
     }
 
     public static final long divisionUsingRecursion(int a, int b) {
+        if (b == 0) {
```

Comment:
Added an if statement that checks if the b integer equals zero. If it does the program will throw an IllegalArgumentException and then display an error message.